text/text.go

```diff
@@ -0,0 +1,62 @@
+package text
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/skuid/skuid/version"
+)
+
+// RunCommand outputs text to be shown at the beginning of each command
+func RunCommand(commandName string) string {
+	return joinError([]string{
+		Separator(),
+		fmt.Sprintf("Skuid CLI Version %v", version.Name),
+		fmt.Sprintf("Running Command: %v", commandName),
+		Separator(),
+	})
+}
+
+// Separator creates some visual space
+func Separator() string {
+	return "------------------------------------------"
+}
+
+// ErrorSeparator creates some visual space
+func ErrorSeparator() string {
+	return "******************************************"
+}
+
+// PrettyError formats an error with a description and message
+func PrettyError(description string, err error) string {
+	return joinError([]string{
+		"",
+		ErrorSeparator(),
+		fmt.Sprintf("%v:", description),
+		err.Error(),
+		Separator(),
+	})
+}
+
+func joinError(errors []string) string {
+	return strings.Join(errors, "\n")
```

Comment:
We may want to see how newlines in our error messages affect the logging in DataDog. In the logstash days, this was problematic. The errors typically need to be single line json for most consumers.

Reply:
I see now that this is skuid-cli, so newlines are A-OK.